pytket/binders/passes.cpp

```diff
@@ -679,6 +679,9 @@ PYBIND11_MODULE(passes, m) {
   m.def(
       "RemoveBarriers", &RemoveBarriers,
       "A pass to remove all barrier instructions from the circuit.");
+  m.def(
+      "RemovePhaseOps", &RemovePhaseOps,
+      "A pass to remove all Phase operations from the circuit.");
```

Comment:
The documentation should probably specify the limits of this pass, i.e. removes conditional Phases but not Phases within other boxes/controlled operations. Beyond this, the PR looks all good